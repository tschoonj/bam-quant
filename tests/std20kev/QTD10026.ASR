File: C:\xrfdaten\2014KW30\chris\std20kev\std10026.spe
Start: 21.07.2014 18:07:13
Stop: 21.07.2014 18:08:53
BESSY: 297.9999532 298.9862314
IONI: 3.756852E-11 3.669989E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 83814
Run: 100000
Dead: 16
ICR: 50600
OCR: 0
Accumulated BESSY: 59662.1044278
Accumulated Ioni: 7.4192226E-09
N Accumulation: 200
Normfactor:  3.2162887e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Fe
exp_x 151.260406494141 151.2604
exp_y 150.323791503906 150.3238
exp_z 53.8899993896484 53.8901
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  26   1      6.399   3597910.      1897.     146.13
