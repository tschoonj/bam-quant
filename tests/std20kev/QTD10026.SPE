$DATA:
0 2000
19654
12387
5323
1637
294
50
7
5
1
3
1
2
1
0
0
0
174
934
1201
1125
1056
1058
918
944
923
921
870
819
807
826
771
747
748
689
715
667
696
672
649
672
673
658
600
610
598
594
633
594
604
609
611
570
558
551
583
508
540
549
564
495
497
521
537
517
518
496
492
519
503
543
503
524
495
488
439
461
483
485
482
494
422
451
445
412
434
446
448
451
445
432
475
483
468
446
470
475
482
505
494
476
484
527
539
514
503
486
444
475
476
466
418
451
410
440
415
450
429
451
431
430
435
434
404
420
420
438
462
467
497
431
450
434
447
397
393
435
438
454
448
428
409
453
427
547
565
604
588
613
581
529
485
455
444
437
443
422
453
463
468
433
440
391
447
443
375
461
405
427
457
432
423
455
439
426
432
440
442
440
443
454
403
414
445
439
454
472
461
478
438
446
479
457
479
449
438
470
460
454
479
458
470
472
479
461
492
509
446
467
538
440
454
491
503
473
485
502
487
507
513
508
565
581
545
662
728
912
1178
1471
1785
2177
2254
2234
1971
1636
1264
932
720
653
560
558
563
539
639
577
602
590
568
577
610
666
633
693
643
705
694
692
721
733
771
786
844
920
998
966
956
964
926
963
956
860
901
849
900
972
856
893
906
919
955
899
931
888
954
976
980
920
996
1017
1026
1020
1097
1176
1201
1315
1302
1425
1438
1576
1647
1789
1950
2090
2293
2589
2962
3573
4769
7039
11911
22850
44814
82452
140379
218969
308053
393135
449249
463405
428369
356132
267592
179496
108964
60160
30126
14134
6505
3251
1972
1533
1387
1290
1303
1374
1396
1541
1519
1867
2170
2683
3633
5312
8306
13358
21344
31784
44461
57654
66650
71861
69225
61486
48879
35261
23287
13845
7574
3854
1832
747
324
165
94
53
37
45
38
34
36
43
30
32
23
28
27
34
26
29
25
21
27
21
24
15
13
20
19
16
26
30
26
22
33
21
20
23
21
13
16
18
19
12
13
13
14
9
15
15
11
17
12
15
13
16
12
15
18
15
15
21
13
13
11
13
14
7
15
23
16
11
14
19
8
9
9
9
16
8
12
17
7
16
12
26
12
12
7
15
13
20
9
16
11
14
15
12
11
12
12
11
12
10
11
13
9
12
10
15
7
13
15
11
15
11
18
10
13
6
12
12
12
13
12
12
18
11
16
9
6
9
12
10
7
15
4
13
9
13
12
6
6
12
11
10
7
8
14
9
12
9
9
5
8
7
14
12
16
11
10
13
8
8
13
11
10
4
10
11
5
8
5
7
14
12
5
6
20
6
8
12
10
21
18
14
19
12
25
17
22
9
12
11
12
12
11
6
7
11
5
11
16
13
8
8
7
11
12
8
14
6
7
10
10
10
11
8
17
6
10
15
13
15
9
11
11
8
15
15
6
8
6
11
11
7
12
9
18
9
8
11
13
7
4
15
7
9
13
13
12
14
17
14
9
17
19
19
11
14
25
19
39
57
58
113
150
238
362
502
699
862
1040
1241
1441
1380
1353
1221
1046
879
691
519
319
219
144
111
61
40
20
13
13
19
15
17
21
21
25
51
69
90
119
149
216
264
277
382
397
454
385
419
352
293
213
196
136
97
49
50
25
9
6
4
6
3
3
2
8
5
3
9
13
9
9
11
20
18
32
32
37
34
27
36
38
21
29
14
15
6
9
5
0
5
1
4
1
3
1
2
1
2
4
3
2
2
2
1
1
1
2
0
1
2
3
2
2
2
0
0
2
1
1
2
1
0
1
2
0
3
3
2
0
0
2
0
1
0
1
3
1
1
1
1
1
1
2
1
2
1
2
3
3
5
8
9
10
13
19
31
24
21
27
32
32
30
29
21
20
13
14
6
7
7
3
1
5
2
2
2
0
3
0
3
0
3
1
1
0
2
0
2
2
2
0
2
2
0
1
2
1
3
0
2
0
2
1
1
1
3
2
1
3
0
4
2
1
2
0
2
1
2
2
2
2
2
0
2
2
1
0
3
4
3
2
2
2
2
2
3
6
1
0
4
3
2
1
2
3
1
3
3
2
4
12
7
12
11
9
13
5
6
7
7
6
7
10
4
6
5
3
6
3
2
2
4
2
5
4
3
1
5
8
6
10
4
5
8
7
7
11
2
9
6
14
10
8
12
12
7
12
15
12
13
14
8
12
12
13
12
18
12
10
10
18
21
13
20
21
13
23
17
24
19
24
20
23
17
32
31
32
38
31
42
29
37
45
37
48
46
45
45
33
52
39
34
42
62
55
54
46
45
54
57
64
55
58
55
63
81
74
69
79
87
96
93
130
110
127
136
124
122
129
158
142
124
138
133
113
106
90
91
85
80
71
44
40
43
40
24
20
21
11
10
7
5
8
6
9
2
1
1
2
1
0
1
1
0
1
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
1
0
0
0
0
0
0
0
1
0
0
0
0
1
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
1
1
0
0
0
1
0
0
1
0
0
0
1
1
0
0
0
1
0
0
0
2
0
0
1
0
1
2
0
1
2
1
2
0
2
0
1
2
0
0
0
0
2
0
2
0
0
2
0
0
3
0
2
2
1
2
1
3
1
0
1
0
1
0
1
1
0
1
0
0
1
1
0
1
0
0
0
0
1
0
1
0
0
0
0
0
0
1
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std20kev\std10026.spe
Start: 21.07.2014 18:07:13
Stop: 21.07.2014 18:08:53
BESSY: 297.9999532 298.9862314
IONI: 3.756852E-11 3.669989E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 83814
Run: 100000
Dead: 16
ICR: 50600
OCR: 0
Accumulated BESSY: 59662.1044278
Accumulated Ioni: 7.4192226E-09
N Accumulation: 200
Normfactor:  3.2162887e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Fe
 
exp_x 151.260406494141 151.2604
exp_y 150.323791503906 150.3238
exp_z 53.8899993896484 53.8901
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  26   1      6.399   3597910.      1897.     146.13
