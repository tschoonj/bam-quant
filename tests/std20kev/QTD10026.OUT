     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:23:40  บ   
     บ  Spectrum: QTD10026.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 100 -1980        4 iterations done    บ   
     บ  ChiSquare =    93.1   last change =   .06%    lambda= 1.E-06   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -54.4 ๑   100.0   บ     -54.4 ๑      .6   บ   
     บ  GAIN (eV/ch)  บ     20.312 ๑   2.000   บ    20.312 ๑    .002   บ   
     บ det NOISE (eV) บ      -47.4 ๑    40.0   บ     -47.4 ๑    -5.7   บ   
     บ  FANO factor   บ       .182 ๑    .050   บ      .182 ๑    .003   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Fe-Ka                           3597910. ๑   1897.           บ   
     บ    KA1         6.399   1.00000     3597910. ๑   1897.    146.1  บ   
     บ              317.720    150.49         202.           9.17E-01  บ   
     บ                                                                 บ   
     บ  2 Fe-Kb                            589163. ๑    768.           บ   
     บ    KB1         7.059   1.00000      589163. ๑    768.     74.8  บ   
     บ              350.214    158.79         185.           9.38E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                           18250. ๑    139.           บ   
     บ    L1N3       12.798    .73811       13471. ๑    103.     61.7  บ   
     บ              632.760    218.04         103.           9.85E-01  บ   
     บ    L1N2       13.458    .24204        4417. ๑     35.     10.5  บ   
     บ              665.254    223.85          90.           9.87E-01  บ   
     บ    L1O3       14.118    .01984         362. ๑      9.      1.8  บ   
     บ              697.747    229.52          78.           9.87E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
